src/Utils/protected/Utils/MgxNumeric.h

```diff
@@ -70,9 +70,17 @@ namespace Math
 
 		static inline bool isNearlyEqual(const double a, const double b)
 		{
-			double s = std::fabs(a) + std::abs(b);
-			double d = a - b;
-			return (d==0.0) ? true : isNearlyZero(d/s);
+            if(a == b) {
+                return true;
+            } else {
+                double d = std::abs(a - b);
+                if (a == 0.0 || b == 0.0) {
+                    return isNearlyZero(d);
```

Comment:
The goal here is to deal with an absolute error, but it can be useful if `a` and `b` are close to zero, not necessarily equal to `0`.